src/core/lib/iomgr/endpoint_pair_windows.cc

```diff
@@ -63,8 +63,16 @@ static void create_sockets(SOCKET sv[2]) {
   GPR_ASSERT(svr_sock != INVALID_SOCKET);
 
   closesocket(lst_sock);
-  grpc_tcp_prepare_socket(cli_sock);
-  grpc_tcp_prepare_socket(svr_sock);
+  grpc_error_handle error = grpc_tcp_prepare_socket(cli_sock);
+  if (!error.ok()) {
+    gpr_log(GPR_WARN, "Prepare cli_sock failed with error: %s",
```

Comment:
gRPC does not have `GPR_WARN`, maybe it could be useful to have one, but for now I suggest to just use `GPR_INFO`: https://github.com/grpc/grpc/blob/67f364e23e4989c6e7837d03ea4e9d415f3cd890/include/grpc/support/log.h#L53-L55

Reply:
Dang, I assumed it would look more like glog so I didn't check. Switched both to GPR_INFO